milestone-02/express-stripe/app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
+const stripe = require('stripe')("");
+
+stripe.charges.create({
```

Comment:
I think the idea is that you submit card information through a form and then call 'charges.create' inside of a 'app.post(...)' entry. You can get the submitted token using req.body.stripeToken (you will also have to install the module 'body-parser'.